### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by HelenBurge**
Adding a note to about us for the intended use of the code in the tests.

Added 2 lines

Closes issue(s):

- N/A

Need for Call for Review:
This can be merged with 1 approval reason: editorial changes to website
---

### **When creating PR:**

- [ ] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [ ] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [ ] Add yourself (and co-authors) as "Assignees" for PR.
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [ ] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: pages/about.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The ACT Rules Community Group is an open forum set up to document and harmonize
...
 
-ACT-R is not set up to remove differences or impose changes on accessibility testing tools and methodologies. There is value in innovation and diverse approaches. Rather it aims to contribute to more consistent results, regardless of how the testing is done. Knowing when something meets a requirement, and when it does not, should be clear and consistent.
+ACT-R is not set up to remove differences or impose changes on accessibility testing tools and methodologies. There is value in innovation and diverse approaches. Rather it aims to contribute to more consistent results, regardless of how the testing is done. Knowing when something meets a requirement, and when it does not, should be clear and consistent. The tests provide code examples that are intended to show a pass/fail or not applicable test. They are not intended to be used as how to code.
```
Comment (Jym77) [inline pages/about.md]:
```suggestion
ACT-R is not set up to remove differences or impose changes on accessibility testing tools and methodologies. There is value in innovation and diverse approaches. Rather it aims to contribute to more consistent results, regardless of how the testing is done. Knowing when something meets a requirement, and when it does not, should be clear and consistent. The tests provide code examples that are intended to show a minimal pass/fail or not applicable test. They are not intended to be showcase "good" HTML code.
```

**Review by Jym77 — CHANGES_REQUESTED** [review]
Good point to add.

File: pages/about.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ The ACT Rules Community Group is an open forum set up to document and harmonize
...
 
-ACT-R is not set up to remove differences or impose changes on accessibility testing tools and methodologies. There is value in innovation and diverse approaches. Rather it aims to contribute to more consistent results, regardless of how the testing is done. Knowing when something meets a requirement, and when it does not, should be clear and consistent.
+ACT-R is not set up to remove differences or impose changes on accessibility testing tools and methodologies. There is value in innovation and diverse approaches. Rather it aims to contribute to more consistent results, regardless of how the testing is done. Knowing when something meets a requirement, and when it does not, should be clear and consistent. The tests provide code examples that are intended to show a minimal pass/fail or not applicable test. They are not intended to be showcase "good" HTML code.
```
Comment (Jym77) [inline pages/about.md]:
```suggestion
ACT-R is not set up to remove differences or impose changes on accessibility testing tools and methodologies. There is value in innovation and diverse approaches. Rather it aims to contribute to more consistent results, regardless of how the testing is done. Knowing when something meets a requirement, and when it does not, should be clear and consistent. The tests provide code examples that are intended to show a minimal pass/fail or not applicable test. They are not intended to showcase "good" HTML code.
```

**Review by Jym77 — CHANGES_REQUESTED** [review]
🙈 We left a typo

File: pages/about.md (hunk 2 above)
Comment (HelenBurge) [reply pages/about.md]: Ack - too early for my brain! Thanks :)

**Review by Jym77 — APPROVED** [review]

**Review by WilcoFiers — APPROVED** [review]

**Review by carlosapaduarte — APPROVED** [review]

**Pull request merged**